test/cluster_test.go

```diff
@@ -124,6 +129,92 @@ func ScaleDownWithMigrateFillDelay(ctx goctx.Context) {
 	)
 }
 
+func clusterWithIgnorePodList(ctx goctx.Context) {
```

Comment:
can we add a test after removing the name from list and make that pod running to check if updated config is correctly reflected on that pod as well?